packages/app/src/elements/items-list.ts

```diff
@@ -37,6 +37,9 @@ export interface ItemsFilter {
     attachments?: boolean;
     recent?: boolean;
     host?: boolean;
+    auditWeak?: boolean;
+    auditReused?: boolean;
+    auditCompromised?: boolean;
```

Comment:
These are the 3 filters I plan on adding, to allow just routing to `/items` and display them.

Reply:
I had envisioned simply listing the items in the audit page but I like this idea!